features/admin.console-settings.v1/components/console-administrators/administrators-list/administrators-list.tsx

```diff
@@ -129,7 +166,14 @@ const AdministratorsList: React.FunctionComponent<AdministratorsListProps> = (
     const [ searchQuery, setSearchQuery ] = useState<string>("");
     const [ showAddExistingUserWizard, setShowAddExistingUserWizard ] = useState<boolean>(false);
     const [ showInviteNewAdministratorModal, setShowInviteNewAdministratorModal ] = useState<boolean>(false);
-    const [ selectedUserStore, setSelectedUserStore ] = useState<string>(userstoresConfig?.primaryUserstoreName);
+    const [ isEnterpriseLoginEnabled, setIsEnterpriseLoginEnabled ] = useState<boolean>(false);
+
+    const [ showAddExternalAdminWizard, setShowAddExternalAdminWizard ] = useState(false);
+    const [ selectedUserStore, setSelectedUserStore ] = useState<string>(
+        isPrivilegedUsersInConsoleSettingsEnabled
+            ? undefined
+            : PRIMARY_USERSTORE
```

Comment:
Use `userstoresConfig?.primaryUserstoreName` instead of the String constant. Check in other places as well.

Reply:
Fixed with [d587fe7](https://github.com/wso2/identity-apps/pull/6938/commits/d587fe7b7259c4a7ddb0ce2652dc1c38c1278380)